CMakeLists.txt

```diff
@@ -293,6 +273,25 @@ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/omplConfig.cmake
     ${CMAKE_CURRENT_BINARY_DIR}/omplConfigVersion.cmake
     DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/ompl/cmake
     COMPONENT ompl)
+install(TARGETS ompl
+    EXPORT omplExport
+    DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    COMPONENT ompl)
+install(TARGETS ompl_app_base
+    EXPORT omplExport
+    DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    COMPONENT omplapp)
+if(TARGET ompl_app)
+    install(TARGETS ompl_app
+        EXPORT omplExport
+        DESTINATION ${CMAKE_INSTALL_LIBDIR}
+        COMPONENT omplapp)
+endif()
+install(EXPORT omplExport
```

Comment:
Will this conflict with the other export? Usually, I've seen `${PROJECT_NAME}Export` to prevent conflicts.